packages/loading-screen/package.json

```diff
@@ -4,9 +4,11 @@
   "license": "AGPL-3.0",
   "scripts": {
     "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build --no-module && yarn zip",
+    "build": "yarn run compile-messages && vue-cli-service build --no-module && yarn zip",
     "lint": "vue-cli-service lint",
-    "zip": "../../scripts/bundle_zip.py loading-screen.zip"
+    "zip": "../../scripts/bundle_zip.py loading-screen.zip",
+    "extract-messages": "formatjs extract 'src/**/*.{js,vue}' --format transifex --out-file lang/en-US.json",
+    "compile-messages": "formatjs compile lang/en-US.json --format transifex --out-file compiled-lang/en-US.json"
```

Comment:
I believe this one can be simplified as `formatjs compile-folder --format transifex lang compiled-lang`. It seems to generate `compiled-lang/*.json` correctly for me. I don't see an equivalent for the extract side, though.

Reply:
Ah excellent! I was looking for a command like that, but obviously didn’t look hard enough. I’ll rework this.

The extract side doesn’t need an equivalent: it already has the glob pattern for selecting all JS and Vue files, and it only ever needs to generate `en-US.json`. We then treat that as equivalent to a gettext `pot` file — the template for all other language catalog files.